### PR TITLE
[DOC] Updated RapidsConf to reflect the default value of `spark.rapids.sql.improvedFloatOps.enabled` [skip ci]

### PR DESCRIPTION
**Pull request by razajafri**
Since the `spark.rapids.sql.improvedFloatOps.enabled` is not disabled by default. 

This PR updates the RapidsConf to reflect that.

Signed-off-by: Raza Jafri <rjafri@nvidia.com>

### CODE REVIEW

**Review by abellina — APPROVED** [review]
LGTM

**Comment by tgravescs** [comment]
please put better description then updated docs

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala [hunk 1]
```diff
@@ -575,8 +575,7 @@ object RapidsConf {
   val IMPROVED_FLOAT_OPS = conf("spark.rapids.sql.improvedFloatOps.enabled")
     .doc("For some floating point operations spark uses one way to compute the value " +
       "and the underlying cudf implementation can use an improved algorithm. " +
-      "In some cases this can result in cudf producing an answer when spark overflows. " +
-      "Because this is not as compatible with spark, we have it disabled by default.")
+      "In some cases this can result in cudf producing an answer when spark overflows.")
     .booleanConf
     .createWithDefault(true)
```
Comment (tgravescs) [inline sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala:580]: we intentionally put the confs into our integration test with the defaults they had, if we change the default here those tests I believe should be updated as well...

integration_tests/src/main/python/spark_session.py:
    'spark.rapids.sql.improvedFloatOps.enabled': 'false',

Can you please file a 22.08 issue to update those tests. We may want to check other defualtas as well

**Comment by razajafri** [comment]
build

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala (hunk 1 above)
Comment (razajafri) [reply sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala:580]: created issue #5844

**Pull request merged**